src/tests/components/ShareLink.test.jsx

```diff
@@ -4,20 +4,26 @@ import { unmountComponentAtNode } from 'react-dom';
 import { act } from 'react-dom/test-utils';
 import ShareLink from '../../components/ShareLink';
 import * as bitlyWorker from '../../utils/BitlyWorker';
-import Zlib from 'browserify-zlib';
 
 // Mock copy to clipboard since we don't need to test the component itself
-jest.mock('copy-to-clipboard', () => {
-  return jest.fn();
+vi.mock('copy-to-clipboard', () => ({
+  default: vi.fn()
+}));
+const { deflateSpy } = vi.hoisted(() => {
```

Comment:
`vi.hoisted`! Intriguing!

Reply:
This was the only way I could figure out how to mock `browserify-zlib` and have access to the mocked function to check what it was called with. Because the vi.mock gets lifted to the top of the file, I had to use the vi.hoisted to make sure the `deflateSpy` mock was available before the vi.mock is created. If you know of a better way to do this, I'm happy to change it.